test/v2/createPool/composableStable.integration.test.ts

```diff
@@ -18,14 +18,20 @@ import { ANVIL_NETWORKS, startFork } from '../../anvil/anvil-global-setup';
 import { doCreatePool } from '../../lib/utils/createPoolHelper';
 import { CreatePoolTxInput } from '../../lib/utils/types';
 
-const { rpcUrl } = await startFork(ANVIL_NETWORKS.MAINNET);
-
 describe('Create Composable Stable Pool tests', () => {
     const chainId = ChainId.MAINNET;
     let txInput: CreatePoolTxInput;
     let poolAddress: Address;
     let createPoolComposableStableInput: CreatePoolV2ComposableStableInput;
+    let rpcUrl: string;
+
     beforeAll(async () => {
+        // TODO: find out why findEventInReceiptLogs doesn't find the event when blockNumber is updated to 21373640n (this causes test to fail)
+        ({ rpcUrl } = await startFork(
+            ANVIL_NETWORKS.MAINNET,
+            undefined,
+            18980070n,
+        ));
```

Comment:
`BalancerErrors.sol` looks like fun :gift: 
https://www.tdly.co/shared/simulation/85d1f5ff-5bd8-4bfb-9a20-d14eefebde6d

Reply:
Nice catch! 
The factory for CSPv5 has been disabled 👀 
Luiz was the one that was keeping track of factory updates and I missed this - I'll bump to CSPv6 over here.
And I'll assume we won't have a newer version on Balancer v2, so we should be ok with that.
But we'll need to keep that in mind to make sure we don't lag behind similar updates for Balancer v3 pools.

Reply:
I created an issue to keep track of this: https://github.com/balancer/b-sdk/issues/535

Reply:
Does that mean we can remove the TODO?